### PR TITLE
Update website

### PR DESCRIPTION
**Pull request by marwin89**
### What does this PR do?

1. update infos and texts (see screenshots)
2. add overview section for overview graphics,
3. add overview graphics under images (for linking to it)
4. improve layout
5. make "source" a seperate section
6. prepare a hidden "new features"-section (in case for a release)
7. Add vocaluxe to "together-with" (cause they are basically in if they get devs)
8. Removed "contributors" in images-folder (not needed)
9. Fix Favicon

### Closes Issue(s)
-

### Motivation

Keep website updated, improve layout

### More
website-update-deprecated
website-update-overview
website-update-sources
website-update-together-with
website-update-txt
new-features

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -108,26 +107,47 @@ <h2 class="cta-heading text-white">UltraStar Format Specification (draft) (offic
...
-                                    <!--<a class="btn btn-white border-none btn-rounded-corner" href="https://github.com/UltraStar-Deluxe/format">Go To GitHub<span class="icon-on-button"><i class="ion-ios-arrow-thin-right"></i></span></a>-->
                                 </div>
                             </div>
                         </div>
                     </div>
                 </div>
             </div>
-
-
+            <!--
```
Comment (Baklap4) [inline docs/index.html:127]: Please add a comment saying why we need this and when we want to remove it :) (after the `<!-- your comment`

File: docs/index.html [hunk 2]
```diff
@@ -2802,7 +2914,7 @@ <h4>Tutorials</h4>
             <hr class="footer-divider">
             <div class="copyright-cta">
                 <p class="text-uppercase">LazyFox Design - All rights Reserved By <a
-                        href="https://themewagon.com/"><span class="span text-violat ">Themewagon</span></a></p>
+                        href="https://themewagon.com/"><span class="span text-violat ">Themewagon</span></a>, big thanks to: bohning, asgard sings, rakuri, marwin89</p>
```
Comment (Baklap4) [inline docs/index.html:2917]: I'd link to our main github profile pages
```
<a href="https://github.com/bohning">bohning</a>
<a href="https://github.com/baklap4">Asgard Sings!</a>
<a href="https://github.com/rakuri255">rakuri255</a>
<a href="https://github.com/marwin89">marwin89</a>
```

**Review by Baklap4 — CHANGES_REQUESTED** [review]
Sorry for the late response, got a couple of items 👍 thanks again for the update!

File: docs/index.html (hunk 1 above)
Comment (marwin89) [reply docs/index.html:127]: done: comment added: this section has to be released when we release format version 1.1.0

File: docs/index.html (hunk 2 above)
Comment (marwin89) [reply docs/index.html]: done: links added

**Comment by marwin89** [comment]
@Baklap4 conflicts are resolved, review requests done 👍

**Comment by marwin89** [comment]
## Commit 6: Code Fixes

- fixed line endings
- checked with html validator (duplicated ids solved etc.)
- fixed favicon - now displayed

**Review by Baklap4 — DISMISSED** [review]
LGTM, but still needs to be rebased correctly
```
git checkout main
git pull
git checkout update-website
git rebase -i origin/main
#fix any conflicts (if conflict -> git add -A && git rebase --continue  )
git push -f # this is destructive so please make sure it's working correctly as before :)
```

**Comment by marwin89** [comment]
@Baklap4 can you review again?
I force-pushed (Using GitHub Desktop cause I'm unexperienced don't use terminal) hope this is right.
All files look correct, website works fine :)

**Comment by marwin89** [comment]
@Baklap4 : could you review this pr? 👋 😃

**Review by Baklap4 — APPROVED** [review]

**Pull request merged**